### PR TITLE
ACE & RHS Compat Expansion

### PR DESCRIPTION
**Pull request by sargken**
https://forums.bistudio.com/topic/185587-ace3xrhs-fragmentation-configs/ all credit goes here. I have Permission from author.

### CODE REVIEW

**Comment by MikeMatrix** [comment]
Please make sure you've read the [ACE3 Coding Guidlines](http://ace3mod.com/wiki/development/coding-guidelines.html) and convert all tabs in this PR to space indentation.

**Comment by sargken** [comment]
Ok ill work on it tomorrow.

File: optionals/compat_rhs_afrf3/CfgAmmo.hpp [hunk 1]
```diff
@@ -126,4 +126,68 @@ class CfgAmmo
...
+	};
+	class rhs_ammo_rgn_exp: rhs_ammo_rgn_base {
+
+	};
+	class rhs_ammo_fakel: GrenadeHand {
+		ace_frag_enabled = 0;
+		ace_frag_skip = 1;
+		ace_frag_force = 0;
+	};
+	class rhs_ammo_fakels: rhs_ammo_fakel{};
```
Comment (Glowbal) [inline optionals/compat_rhs_afrf3/CfgAmmo.hpp]: Missing space in front of `{}`

File: optionals/compat_rhs_usf3/CfgAmmo.hpp [hunk 2]
```diff
@@ -122,4 +122,61 @@ class CfgAmmo
...
     };
+    class GrenadeHand;
+	class rhs_ammo_mk3a2: GrenadeHand
+	{
+		ace_frag_enabled=0;
+		ace_frag_skip=1;
+		ace_frag_force=0;
+	};
+	class rhs_ammo_m84: GrenadeHand
+	{
```
Comment (Glowbal) [inline optionals/compat_rhs_usf3/CfgAmmo.hpp]: Should be on one line: `class rhs_ammo_m84: GrenadeHand {`

File: optionals/compat_rhs_usf3/CfgAmmo.hpp [hunk 3]
```diff
@@ -122,4 +122,61 @@ class CfgAmmo
...
+		ace_frag_enabled=0;
+		ace_frag_skip=1;
```
Comment (Glowbal) [inline optionals/compat_rhs_usf3/CfgAmmo.hpp]: Should have spaces on both sides of the `=`.

Example: `ace_frag_skip = 1;`

**Comment by sargken** [comment]
Im going to add more to this like the gun overheating and adding in more of the RHS weapons.

File: optionals/compat_rhs_afrf3/CfgAmmo.hpp [hunk 4]
```diff
@@ -126,4 +126,66 @@ class CfgAmmo
...
+    class rhs_rpg7v2_tbg7v: rhs_rpg7v2_pg7vl {
+        ace_frag_enabled = 0;
+        ace_frag_skip = 1;
+        ace_frag_force = 0;
+	};
+    class rhs_rshg2_rocket: rhs_rpg7v2_tbg7v {
+        ace_frag_enabled = 0;
+        ace_frag_skip = 1;
+        ace_frag_force = 0;
+	};
```
Comment (commy2) [inline optionals/compat_rhs_afrf3/CfgAmmo.hpp]: tab

File: optionals/compat_rhs_usf3/CfgAmmo.hpp [hunk 5]
```diff
@@ -122,4 +123,48 @@ class CfgAmmo
...
+    class rhs_ammo_mk3a2: GrenadeHand {
+        ace_frag_enabled = 0;
+        ace_frag_skip = 1;
+        ace_frag_force = 0;
+	};
+    class rhs_ammo_m84: GrenadeHand {
+        ace_frag_enabled = 0;
+        ace_frag_skip = 1;
+        ace_frag_force = 0;
+	};
```
Comment (commy2) [inline optionals/compat_rhs_usf3/CfgAmmo.hpp]: Tab

File: optionals/compat_rhs_afrf3/CfgAmmo.hpp [hunk 6]
```diff
@@ -126,4 +126,66 @@ class CfgAmmo
...
+    class RocketBase;
+    class R_PG32V_F: RocketBase {};
+    class rhs_rpg26_rocket: R_PG32V_F {};
+    class rhs_rpg7v2_pg7vl: rhs_rpg26_rocket {};
+    class rhs_rpg7v2_og7v: rhs_rpg7v2_pg7vl {
+        ace_frag_enabled = 1;
+        ace_frag_metal = 400;
+        ace_frag_charge = 210;
+        ace_frag_gurney_c = 2800;
+        ace_frag_gurney_k = 3/5;
```
Comment (commy2) [inline optionals/compat_rhs_afrf3/CfgAmmo.hpp]: Should be "3/5" with quote marks imo. Just to make sure the builder uses the full expression.

**Comment by commy2** [comment]
Is `rhs_ammo_M136_hedp_rocket` etc. really the capitalization they use? I know that it makes no difference for the game, but certain missions / scripts might not take it into acount when using commands like `in`.

File: optionals/compat_rhs_afrf3/CfgAmmo.hpp (hunk 6 above)
Comment (sargken) [inline optionals/compat_rhs_afrf3/CfgAmmo.hpp]: ? What are you referring to?

**Comment by MikeMatrix** [comment]
It probably will be wise to squash the commits before we merge this.

**Comment by sargken** [comment]
How do i do that?

**Comment by MikeMatrix** [comment]
[Read this](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/)

**Comment by alganthe** [comment]
And for the love of god [learn to write a proper commit message](http://chris.beams.io/posts/git-commit/)

**Comment by sargken** [comment]
I'm going to make a cleaner pull request.

**Pull request closed without merging**